app/web/.env.localdev

```diff
@@ -5,8 +5,6 @@ NEXT_PUBLIC_CONSOLE_BASE_URL="http://localhost:10027"
 NEXT_PUBLIC_IS_POST_BETA_ENABLED=true
 NEXT_PUBLIC_MAPBOX_KEY="pk.eyJ1IjoiY291Y2hlcnMiLCJhIjoiY2tpYzY4eHd2MGVpcTJ0bGdhdGhvbHhlbiJ9.u5zvDeFE9H8itTK_dNp7Pg"
 NEXT_PUBLIC_NOMINATIM_URL="https://nominatim.openstreetmap.org/"
-NEXT_PUBLIC_TILE_URL="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
-NEXT_PUBLIC_TILE_ATTRIBUTION="Map data &copy; <a href="https://osm.org/copyright">OpenStreetMap</a> contributors"
```

Comment:
Same here, doesn't seem related to this PR?

Reply:
Moved to #4690